rsa.go

```diff
@@ -49,7 +52,10 @@ func (rv *rsaVerifier) Algorithm() Algorithm {
 //
 // Reference: https://datatracker.ietf.org/doc/html/rfc8152#section-8
 func (rv *rsaVerifier) Verify(digest []byte, signature []byte) error {
-	hash, _ := rv.alg.hashFunc()
+	hash, ok := rv.alg.hashFunc()
+	if !ok {
+		return ErrVerification
```

Comment:
Maybe this could be more specific on the kind of error, i.e., `ErrInvalidAlgorithm`

Reply:
Updated to `ErrInvalidAlgorithm`